### PR TITLE
Update faq.rst

### PR DESCRIPTION
**Pull request by APAwad**

### CODE REVIEW

File: faq.rst [hunk 1]
```diff
@@ -51,3 +51,37 @@ If...
...
 - some rarely needed particular modding capabilities
+
+How can I get capes? / Can I install capes?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, you can get capes; first go to the link `Minecraft Capes Pack <https://www.mediafire.com/file/4je935z6ki94j6v/Capes_1.7%252B_%2528PatarHD%2529.zip/file>`_
+
+After downloading the file, Open it and go to :code:`vanilla`.
+
+All the required files will be there.If you want to add your own custom skin,Put it in the :code:`vanilla` folder but **DO NOT** delete the :code:`nitrosfn.png`skin or the :code:`alex.png`skin.
```
Comment (APAwad) [inline faq.rst]:
```suggestion
All the required files will be there.If you want to add your own custom skin,Put it in the :code:`vanilla` folder but **DO NOT** delete the :code:`nitrosfn.png` skin or the :code:`alex.png` skin.
```

File: faq.rst [hunk 2]
```diff
@@ -51,3 +51,37 @@ If...
...
+How can I get capes? / Can I install capes?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, you can get capes; first go to the link `Minecraft Capes Pack <https://www.mediafire.com/file/4je935z6ki94j6v/Capes_1.7%252B_%2528PatarHD%2529.zip/file>`_
+
+After downloading the file, Open it and go to :code:`vanilla`.
+
+All the required files will be there.If you want to add your own custom skin,Put it in the :code:`vanilla` folder but **DO NOT** delete the :code:`nitrosfn.png` skin or the :code:`alex.png` skin.
+
+If you want to change the cape,go to the other folder :code:`cape presets`and name the cape you want:code:`cape.png`and then place it in the :code:`vanilla`folder and replace the cape that is currently in the folder
```
Comment (APAwad) [inline faq.rst]:
```suggestion
If you want to change the cape,go to the other folder :code:`cape presets` and name the cape you want:code:`cape.png` and then place it in the :code:`vanilla` folder and replace the cape that is currently in the folder
```

File: faq.rst [hunk 3]
```diff
@@ -51,3 +51,37 @@ If...
...
+
+If you want a custom cape, make sure it is a 64x32 pixel cape otherwise it will not work.
+
+Now,in a seperate finder folder, go to the directory :code:`~Library/Application Support/versions` then go to the folder which shows the version of minecraft you run.
+
+After, go to :code:`assets` then go to :code:`skin_packs` then :code:`vanilla`.
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for :code:`
```
Comment (APAwad) [inline faq.rst]:
```suggestion
Open the new :code:`skin.json` file with text edit and look for 
```

File: faq.rst [hunk 4]
```diff
@@ -51,3 +51,37 @@ If...
...
+If you want a custom cape, make sure it is a 64x32 pixel cape otherwise it will not work.
+
+Now,in a seperate finder folder, go to the directory :code:`~Library/Application Support/versions` then go to the folder which shows the version of minecraft you run.
+
+After, go to :code:`assets` then go to :code:`skin_packs` then :code:`vanilla`.
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+{
```
Comment (APAwad) [inline faq.rst]:
```suggestion
:code:`{`
```

File: faq.rst [hunk 5]
```diff
@@ -51,3 +51,37 @@ If...
...
+  "skins": [
+    {
+      "localization_name": "Steve",
+      "geometry": "geometry.humanoid.custom",
```
Comment (APAwad) [inline faq.rst]:
```suggestion
:code:`"geometry": "geometry.humanoid.custom",`
```

File: faq.rst [hunk 6]
```diff
@@ -51,3 +51,37 @@ If...
...
+
+After, go to :code:`assets` then go to :code:`skin_packs` then :code:`vanilla`.
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  "skins": [
+    {
+      "localization_name": "Steve",
```
Comment (APAwad) [inline faq.rst]:
```suggestion
      :code:`"localization_name": "Steve",`
```

File: faq.rst [hunk 7]
```diff
@@ -51,3 +51,37 @@ If...
...
+Now,in a seperate finder folder, go to the directory :code:`~Library/Application Support/versions` then go to the folder which shows the version of minecraft you run.
+
+After, go to :code:`assets` then go to :code:`skin_packs` then :code:`vanilla`.
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  "skins": [
+    {
```
Comment (APAwad) [inline faq.rst]:
```suggestion
   :code:`{`
```

File: faq.rst [hunk 8]
```diff
@@ -51,3 +51,37 @@ If...
...
+
+Now,in a seperate finder folder, go to the directory :code:`~Library/Application Support/versions` then go to the folder which shows the version of minecraft you run.
+
+After, go to :code:`assets` then go to :code:`skin_packs` then :code:`vanilla`.
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  "skins": [
```
Comment (APAwad) [inline faq.rst]:
```suggestion
  :code:`"skins": [`
```

File: faq.rst [hunk 9]
```diff
@@ -51,3 +51,37 @@ If...
...
+
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  :code:`"skins": [`
+   :code:`{`
+      :code:`"localization_name": "Steve",`
+:code:`"geometry": "geometry.humanoid.custom",`
+      "texture": "nitrosfn.png",
```
Comment (APAwad) [inline faq.rst]:
```suggestion
      :code:`"texture": "nitrosfn.png",`
```

File: faq.rst [hunk 10]
```diff
@@ -51,3 +51,37 @@ If...
...
+	  "cape": "cape.png",
+      "type": "free"`.
```
Comment (APAwad) [inline faq.rst]:
```suggestion
      :code:"type": "free"`.`
```

File: faq.rst [hunk 11]
```diff
@@ -51,3 +51,37 @@ If...
...
+From the other finder, where your custom cape and skin is, copy all the files and post it in the :code:`vanilla` where your default minecraft files are and replace the default files with the new ones.
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  :code:`"skins": [`
+   :code:`{`
+      :code:`"localization_name": "Steve",`
+:code:`"geometry": "geometry.humanoid.custom",`
+      :code:`"texture": "nitrosfn.png",`
+	  "cape": "cape.png",
```
Comment (APAwad) [inline faq.rst]:
```suggestion
	  :code:`"cape": "cape.png",`
```

File: faq.rst [hunk 12]
```diff
@@ -51,3 +51,37 @@ If...
...
+
+Open the new :code:`skin.json` file with text edit and look for 
+:code:`{`
+  :code:`"skins": [`
+   :code:`{`
+      :code:`"localization_name": "Steve",`
+:code:`"geometry": "geometry.humanoid.custom",`
+      :code:`"texture": "nitrosfn.png",`
+	  :code:`"cape": "cape.png",`
+      :code:"type": "free"`.`
```
Comment (APAwad) [inline faq.rst]:
```suggestion
      :code:"type": "free"`.
```

Comment (APAwad) [inline faq.rst]:
```suggestion
      :code:`"type": "free"`.`
```

**Comment by ChristopherHX** [comment]
Is it only me or is the embedded json of this PR still looking weird?

On github rst viewer this code looks better.
```
.. code-block:: json

  {
    "skins": [
      {
        "localization_name": "Steve",
        "geometry": "geometry.humanoid.custom",
        "texture": "nitrosfn.png",
        "cape": "cape.png",
        "type": "free"
      }
    ]
  }
```
_I heard custom capes stopped working with 1.16_

**Comment by Mhowser** [comment]
Yep this pr is deprecated unfortunately. Also this was back when we didn't know how to handle editing RST docs lol

**Pull request closed without merging**